website_sale_stock_provisioning_time/views/website_sale_product.xml

```diff
@@ -11,7 +11,12 @@
                 <t t-if="product.provisioning_time_in_workdays">
                     <span>Provisioning time until shipment:</span>
                     <span t-esc="product.provisioning_time_in_workdays" />
-                    Work days
+                    <t t-if="product.provisioning_time_in_workdays == 1">
+                        Work day
```

Comment:
It lacks translation. German: "Arbeitstag"

Reply:
I have added the translation